include/settings/settings.h

```diff
@@ -42,8 +42,11 @@ typedef ssize_t (*settings_read_cb)(void *cb_arg, void *data, size_t len);
  * These are registered using a call to @ref settings_register.
  */
 struct settings_handler {
```

Comment:
Id add a settings_handler_static variant without the node field which is what should be used by the macro when statically defining things so we don't take more space than needed.

Reply:
This could be done, but would have a big influence on the code, now throughout the code there is only one settings_handler type that is shared for static and dynamic handlers and in the case of "bestmatch" comparison this is shared by both static and dynamic handlers. Actually if we split up the settings_handler type we might even be using more ROM (code size) handling two different settings_handler structs than we are saving.